doc/conf.py

```diff
@@ -39,17 +39,28 @@
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.viewcode',
     'sphinxcontrib.bibtex',
-    'edit_on_github'
-    # 'nbsphinx'
+    'edit_on_github',
+    'sphinx_gallery.gen_gallery'
 ]
 
-# nbsphinx settings
-
-exclude_patterns = ['_build', '**.ipynb_checkpoints', 'tutorials/.ipynb_checkpoints', '*-checkpoint.ipynb']
-nbsphinx_execute = 'never'
-nbsphinx_epilog = """
-.. note:: :download:`Click here <../{{env.docname}}.ipynb>` to download this gallery page as an interactive Jupyter notebook.
-"""
+from glob import glob
+import shutil
+import os
+
+sphinx_gallery_conf = {
+    # path to your example scripts
+    'examples_dirs': 'tutorials',
+    # path where to save gallery generated examples
+    'gallery_dirs': 'gallery',
+    # build files that start 'tutorial_' and don't contain 'skip'
+    'filename_pattern': r'tutorial_(?!skip)',
+    # first notebook cell in generated Jupyter notebooks
+    'first_notebook_cell': ("# This cell is added by sphinx-gallery\n"
+                            "# It can be customized to whatever you like\n"
+                            "%matplotlib inline"),
```

Comment:
where will this show up?

Reply:
This will show up as the first cell in the generated Jupyter notebook. I left it here so that we remember to replace it with something more useful - for example, maybe the Xanadu/PennyLane logo, links to documentation/gituhub, boilerplate python statements, etc.

We should do this in a new PR to `doc-refactor` once the layout is finalized.